core/chain_util.go

```diff
@@ -42,6 +42,9 @@ var (
 
 	ExpDiffPeriod = big.NewInt(100000)
 	blockHashPre  = []byte("block-hash-") // [deprecated by eth/63]
+
+	mipmapPre    = []byte("mipmap-bloom-")
```

Comment:
It's not a receipt bloom, it's a log bloom. And add another set of bloom filters for what? Do you mean `mipmap-log-bloom` or something?